camptix.php

```diff
@@ -185,6 +188,18 @@ function schedule_events() {
 	}
 
 	/**
+	 * Load Textdomain
+	 */
+	function load_textdomain() {
+
+		$locale = apply_filters( 'plugin_locale', get_locale(), 'camptix' );
+
+		load_textdomain( 'camptix', WP_LANG_DIR . '/camptix/camptix-' . $locale . '.mo' );
```

Comment:
@juanfra What is your reasoning for including the above two lines in addition to `load_plugin_textdomain` below? [As far as I can tell](http://ottopress.com/2013/language-packs-101-prepwork/), the `WP_LANG_DIR` is where the language pack files get downloaded to, and where WordPress looks for them already.